packages/tables/backend/sirius-components-tables/src/main/java/org/eclipse/sirius/components/tables/descriptions/LineDescription.java

```diff
@@ -83,6 +96,10 @@ public static final class Builder {
 
         private Function<VariableManager, List<Object>> semanticElementsProvider;
 
+        private Function<VariableManager, Integer> initialHeightProvider = variableManager -> 53; // 'comfortable' height by default
+
+        private Predicate<VariableManager> isResizablePredicate = variableManager -> false;
+
```

Comment:
Please remove this and never make any choice for the specifier here. Those provider/predicate should be required (as they are) and without any default values. This is explicitly not the job of these projects to decide things like that.

Reply:
Done